To: vim_dev@googlegroups.com
Subject: Patch 7.3.386
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.386
Problem:    Test 83 fails when iconv does not support cp932. (raf)
Solution:   Test if conversion works. (Yukihiro Nakadaira)
Files:	    src/testdir/test83.in


*** ../vim-7.3.385/src/testdir/test83.in	2011-10-12 19:53:31.000000000 +0200
--- src/testdir/test83.in	2011-12-21 21:01:44.000000000 +0100
***************
*** 2,8 ****
  
  STARTTEST
  :so mbyte.vim
! :if !has('iconv')
  : e! test.ok
  : w! test.out
  : qa!
--- 2,8 ----
  
  STARTTEST
  :so mbyte.vim
! :if !has('iconv') || iconv("\x82\x60", "cp932", "utf-8") != "\uff21"
  : e! test.ok
  : w! test.out
  : qa!
*** ../vim-7.3.385/src/version.c	2011-12-23 14:54:01.000000000 +0100
--- src/version.c	2011-12-23 14:55:38.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     386,
  /**/

-- 
WOMAN:   King of the who?
ARTHUR:  The Britons.
WOMAN:   Who are the Britons?
ARTHUR:  Well, we all are. we're all Britons and I am your king.
                                  The Quest for the Holy Grail (Monty Python)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
